builder/lxd/step_lxd_launch.go

```diff
@@ -0,0 +1,50 @@
+package lxd
+
+import (
+	"fmt"
+	"github.com/mitchellh/multistep"
+	"github.com/mitchellh/packer/packer"
+	"time"
+)
+
+type stepLxdLaunch struct{}
+
+func (s *stepLxdLaunch) Run(state multistep.StateBag) multistep.StepAction {
+	config := state.Get("config").(*Config)
+	ui := state.Get("ui").(packer.Ui)
+
+	name := config.ContainerName
+	image := config.Image
+
+	args := []string{
+		"launch", "--ephemeral=false", image, name,
+	}
+
+	ui.Say("Creating container...")
+	_, err := LXDCommand(args...)
+	if err != nil {
+		err := fmt.Errorf("Error creating container: %s", err)
+		state.Put("error", err)
+		ui.Error(err.Error())
+		return multistep.ActionHalt
+	}
+	// TODO: Should we check `lxc info <container>` for "Running"?
+	// We have to do this so /tmp doens't get cleared and lose our provisioner scripts.
+	time.Sleep(1 * time.Second)
```

Comment:
When possible I'm always in favor of an explicit check instead of a sleep, but I don't think it's necessarily a blocker.

Reply:
Yeah, I wasn't thrilled for this, but I it is a little racy on when `/tmp` gets cleared.